Magic 8-ball/Controllers/SettingsViewController.swift

```diff
@@ -40,7 +40,7 @@ class SettingsViewController: SwipeTableViewController {
         let alert = UIAlertController(title: "Add New Answer", message: "", preferredStyle: .alert)
         let action = UIAlertAction(title: "Add", style: .default) { [weak self] (action) in
             self?.answers.append(textField.text!)
```

Comment:
Стоит избегать force unwrap, плохой тон их использовать, лучше указать дефолтное значение
```suggestion
            self?.answers.append(textField.text ?? "")
```